spec/lib/jobs/install-os-job-spec.js

```diff
@@ -152,6 +152,38 @@ describe('Install OS Job', function () {
         );
     });
 
+    describe('test _validateOptions', function() {
```

Comment:
Lack some negative test cases:
- it should throw AssertionError if `mountPoint` is not specified or not in correct type.
- it should throw AssertionError if `size` is not in correct type.
- it should throw AssertionError if `fsType` is not in correct type.